### DISCUSSION TITLE
Error encountered reading results file

### DISCUSSION THREAD

**Discussion by Roaming-Mars** [snl-dakota/dakota#30] [Q&A]
I have a multidim parameter study written that ran to completion and now I am receiving an error when trying to rerun with different partition values. It is "error (s) encountered reading results file <file location>. Item "Based" found while reading function values is not a valid floating point number." Do you have a recommendation on how I debug this issue? Thank you for your help!

**Comment by jadamstephens** [comment]
That's a formatting problem with your results file. It probably indicates that there's a problem with your driver (the driver is writing a label for a response without the value). This is a common problem with drivers written in Bash. The driver will contain something like:

``` bash
echo $VALUE Based >> $results
```
And due to a problem that occurred earlier in the in the run of the driver, the variable `VALUE` doesn't contain anything. So, the results file just has the label 'Based' on a line by itself.
